apps/scan/frontend/src/app_root.tsx

```diff
@@ -36,28 +31,25 @@ import {
   getPollsInfo,
   getScannerStatus,
   getUsbDriveStatus,
+  systemCallApi,
 } from './api';
 import { VoterScreen } from './screens/voter_screen';
 import { LoginPromptScreen } from './screens/login_prompt_screen';
 import { CastVoteRecordSyncRequiredScreen } from './screens/cast_vote_record_sync_required_screen';
 import { SystemAdministratorScreen } from './screens/system_administrator_screen';
 
 export interface Props {
-  hardware: Hardware;
   logger: Logger;
 }
 
-export function AppRoot({ hardware, logger }: Props): JSX.Element | null {
+export function AppRoot({ logger }: Props): JSX.Element | null {
   const authStatusQuery = getAuthStatus.useQuery();
   const configQuery = getConfig.useQuery();
   const pollsInfoQuery = getPollsInfo.useQuery();
   const usbDriveStatusQuery = getUsbDriveStatus.useQuery();
   const checkPinMutation = checkPin.useMutation();
 
-  const { computer } = useDevices({
-    hardware,
-    logger,
-  });
+  const batteryInfoQuery = systemCallApi.getBatteryInfo.useQuery();
```

Comment:
Previously we had the very light `computer` abstraction which was really just a wrapper / flags derived from the batteryInfo. I think it's fine to just use the raw battery info returned from the backend, and the frontends decide what they want to do with it.